### ISSUE TITLE
biopieces broken?

### ISSUE THREAD

**Issue by apjanke** [Homebrew/homebrew-science#4067]
### Indicate troubleshooting steps taken

- [X] Ran `brew update` and retried your prior step?
- [X] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [X] Checked that a similar issue hasn't already been filed?
- [ ] Looked up the software bug tracker if reporting a build error?
- [ ] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?

----------------

The `biopieces` formula looks broken in a couple ways. It depends on a Ruby gem that appears no longer available. And it's pointing to a dead Google Code site, https://code.google.com/p/biopieces/.

```
$ brew install biopieces
==> Installing biopieces from homebrew/science
biopieces: Unsatisfied dependency: biopieces
Homebrew does not provide special Ruby dependencies; install with:
  `gem install biopieces`
Error: Unsatisfied requirements failed this build.
[✘ /usr/local/Library/Taps/homebrew/homebrew-science on ⇄ master]
$ gem install biopieces
ERROR:  Could not find a valid gem 'biopieces' (>= 0) in any repository
ERROR:  Possible alternatives: bio-plates, bits-n-pieces, pieces, time_pieces
```

There's now a GitHub repo, https://github.com/maasha/biopieces, that the formula could be updated to use, though there are no releases in that repo so it'd be head-only or tagged to commits.

Is anyone actually using biopieces? Worth the effort to update, or should it be boneyarded? CC @sjackman as the formula's initial author.

**Comment by sjackman** [comment]
I've created an upstream issue https://github.com/maasha/biopieces/issues/5 asking the author to tag a stable release. If he's responsive, let's update it, if not, let's boneyard it.

**Comment by sjackman** [comment]
Upstream has released `biopieces` 2.0. Let's update to that.

**Comment by sjackman** [comment]
Fixed by #4070

**Comment by apjanke** [comment]
LGTM. Thanks!
